.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+<!-- DO NOT DELETE THE TEXT BELOW. Please make sure relevant boxes are checked [x] -->
+
+Thank you for taking the time to work on a PR for Freeside's Student Resources!
+
+To ensure your PR is dealt with swiftly please check the following:
+
+
+- [ ] Additions are not already listed at any of the [pages](https://github.com/FreesideHull/StudentResources/tree/main/Pages) or README
+- [ ] You have searched the repository for any relevant [issues](https://github.com/FreesideHull/StudentResources/issues) or [PRs](https://github.com/FreesideHull/StudentResources/pulls), including closed ones for any reasons why your entry/entries may be denied. If in doubt: get in touch at our [Discord](https://discord.freeside.co.uk)
+- [ ] Any category you are creating has the minimum requirement of 3 items.
+  If not, your addition may be inserted into `Other interesting stuff `.
```

Comment:
Perhaps this will read better?
```suggestion
- [ ] Any new category created has the minimum requirement of 3 items.
  If not, your addition must be inserted into `Other interesting stuff ` instead.
```